cider/common/hashtable/FixedHashSet.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ * Copyright (c) 2016-2022 ClickHouse, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#pragma once
+
+#include <common/hashtable/FixedHashTable.h>
+
+template <typename Key, typename Allocator = HashTableAllocator>
+class FixedHashSet

```

Comment:
do we need this class?

Reply:
Yes, it may be used in `Aggregator` to store all inserted keys.

Reply:
Please provide some examples for how to use `Aggregator` if necessary

Reply:
Examples will be available in PR for [POAE7-2678](https://jira.devtools.intel.com/browse/POAE7-2678)